Mathlib/Algebra/Order/BigOperators/Group/Finset.lean

```diff
@@ -387,6 +400,12 @@ theorem prod_le_prod_of_ne_one' (h : ∀ x ∈ s, f x ≠ 1 → x ∈ t) :
         (prod_le_one' <| by simp only [mem_filter, and_imp]; exact fun _ _ ↦ le_of_eq)
         (prod_le_prod_of_subset' <| by simpa only [subset_iff, mem_filter, and_imp])
 
+omit [IsOrderedMonoid M] in
```

Comment:
Let's do this in a future PR, it looks like you could make this change to every lemma in this section